src/main/java/org/jenkinsci/plugins/electricflow/action/CloudBeesCDPBABuildDetails.java

```diff
@@ -8,8 +8,10 @@
 import org.jenkinsci.plugins.electricflow.causes.EFCause;
 import org.jenkinsci.plugins.electricflow.models.CIBuildDetail;
 import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
 
 // TODO: Since we have now 2 classes that are doing pretty match the same
+@ExportedBean
```

Comment:
`@ExportedBean` is required for the `@Exported` annotations on the fields in this class to show up in the Jenkins REST API. Without `@ExportedBean`, those fields are not exposed.